game/src/constants.js

```diff
@@ -2,11 +2,17 @@ const gravity = new BABYLON.Vector2(0, -9.8);
 const flightForce = 5; // The amount of force applied when the "fly" button is pressed
 
 const greetings = [
-    'It\'s a bird!',
-    'It\'s a plane!',
-    '1000% more Cube!',
-    'A Whole New Dimension!',
-    'Hello! It\'s Ryan, Coco, William, Terrie！！！',
-    'some changes!',
-    'Just breaking things :)'
+  "It's a bird!",
```

Comment:
Can you change these quotes back to be single quotes? You can use `\'` to escape quotes in strings.

Reply:
sorry, turned prettier off